lib/db/mysql.js

```diff
@@ -554,6 +554,48 @@ module.exports = function (log, error) {
     return this.readFirstResult(GET_UNLOCK_CODE, [uid])
   }
 
+  // METRICS
+
+  // Select : accounts
+  // Where  : createdAt < $1
+  var COUNT_ACCOUNTS_CREATED_BEFORE = 'call countAccountsCreatedBefore_1(?)'
+
+  MySql.prototype.countAccountsCreatedBefore = function (createdBefore) {
```

Comment:
> ...I think we can remove the `createdBefore` parameter for this query and the one below.

Not only did I think wrong, we actually need to introduce a `createdBefore` parameter for the multi-device queries too.

The date-handling code in the Heka filter assumes the data is from the preceding midnight and this is also how the graphs themselves represent the information. But more importantly, if I'd thought it through properly I'd have realised that not using a `createdBefore` parameter actually violates data integrity. Without it, any slippage in the time when the query is run would cause each "day" to own a greater or lesser slice of time than its peers.